src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS OTel .NET SDK for Lambda
+
+This repo contains SDK to instrument Lambda handler to create incoming span.
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Contrib.Instrumentation.AWSLambda
+```
+
+## Configuration
+
+Add `AddLambdaConfigurations()` to `TracerProvider`.
+
+```csharp
+TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
+        // add other instrumentations
+        .AddLambdaConfigurations()
+        .Build();
+```
+
+## Instrumentation
+
+### Lambda Function
+
+1. Create a function with the same signature as the original Lambda function. Call
+`AWSLambdaWrapper.Trace()` API and pass `TracerProvider`, original Lambda function
+and its inputs as parameters.
+
+2. Set the Lambda handler field point to the new function.
```

Comment:
Is the example below for point 1? If so, we should move it to that point.
Also, rephrase point 2 to something like "In your lambda function configuration, set this wrapper function as the lambda handler." Add an example or screenshot if possible.

Reply:
For each component, it indicates steps one by one to do the configuration or instrumentation for using Lambda handler or Lambda handler hosting Asp.Net Core serverless application. And the sample below simply shows a complete steps to include configurations and instrumentation for a simple Lambda function. If we put it in step 1, seems a little bit duplicated and we may have to provide an example for the AspNet Core as well which I think makes the doc even more verbose. Will rephrase the point 2.